.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Enhancement Request
```

Comment:
this file doesn't seems to exist in other repo's PRs.. can you look at cstor PR?

Reply:
Other repos named it as `feature_request.md` basically I took it from Kubernetes anyways I will update it

Reply:
Updated the file name to `feature_request.md`